templates/Header/SideButton/SideButton.jsx

```diff
@@ -0,0 +1,14 @@
+import { Grid } from '@mui/material'
+import React from 'react'
+import headerStyles from '../../../styles/Header/styles' 
+
+const SideButton = ({icon, number, background, borderColor}) => {
+  return (
+    <Grid container item sx={{...headerStyles.sideButton, backgroundColor:{background}, borderColor:{borderColor}}}>
```

Comment:
`
<Grid {...headerStyles.sideButton(background, number, borderColor)}
`
In your styles file

`
sideButton: (background, number, borderColor) => ({
bgcolor: background,
borderColor,
})
`